### PR TITLE
Barbieri1982 week 08

### PR DESCRIPTION
**Pull request by barbieri1982**
multiple fix
some minor fix some error fix

### CODE REVIEW

File: pages/web/domains/api_domain_order/guide.it-it.md [hunk 1]
```diff
@@ -109,9 +109,10 @@ The second step consists in fetching available offers for a domain name.
 >
 > @api {GET} /order/cart/{cartId}/domain
 
-| Parameter | Required | Default | Description         |
-| --------- | -------- | ------- | ------------------- |
-| domain    | yes      | ""      | Desired domain name |
+| Parameter | Required | Default | Description                           |
+| --------- | -------- | ------- | ------------------------------------- |
+| cartId    | yes      | ""      | cartId received from the last command |
+| domain    | yes      | ""      | Desired domain name                   |
```
Comment (benoitmasson) [inline pages/web/domains/api_domain_order/guide.it-it.md:115]: maybe add a `(body)` item to indicate that this parameter should be given in the request payload
see e.g. https://docs.ovh.com/gb/en/domains/api-order/#add-configuration

File: pages/web/domains/api_domain_order/guide.it-it.md [hunk 2]
```diff
@@ -668,8 +669,8 @@ This call also validates the configurations, like owner eligibility for a domain
 
 | Parameter                           | Required | Default | Description                                                                            |
 | ----------------------------------- | -------- | ------- | -------------------------------------------------------------------------------------- |
-| `autoPayWithPreferredPaymentMethod` | yes      | ""      | Used to automatically pay the purchase order with de default payment method of the nic |
-| `waiveRetractationPeriod`           | yes      | ""      | Required for a domain name, it is used to waive your right to retract                  |
+| `autoPayWithPreferredPaymentMethod` | no      | ""      | Used to automatically pay the purchase order with de default payment method of the nic |
+| `waiveRetractationPeriod`           | no      | ""      | Required for a domain name, it is used to waive your right to retract                  |
```
Comment (benoitmasson) [inline pages/web/domains/api_domain_order/guide.it-it.md:673]: description says it is required, are you sure about your change? If so, maybe adjust description

**Review by benoitmasson — APPROVED** [review]

File: pages/web/domains/api_domain_order/guide.it-it.md (hunk 2 above)
Comment (barbieri1982) [reply pages/web/domains/api_domain_order/guide.it-it.md:673]: Hi, 100% sure. Here the proof: the order 186056029 was created by API.

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Comment by OVH-Team-Guides** [comment]
Hello @barbieri1982, I'm closing the PR as it's become unmergeable.
Most of the updates have been recycled into another PR. I'm not deleting your branch so you consider posting the remaining updates onto a new rebased branch. Thanks for your contribution.

**Pull request closed without merging**